conda_lock/src_parser/poetry.py

```diff
@@ -32,15 +32,19 @@ def join_version_components(pieces):
 
 def normalize_pypi_name(name: str) -> str:
     if name in get_lookup():
-        return get_lookup()[name]["conda_forge"]
+        lookup = get_lookup()[name]
+        return lookup.get("conda_name") or lookup.get("conda_forge")
     else:
         return name
 
 
 def poetry_version_to_conda_version(version_string):
     components = [c.replace(" ", "").strip() for c in version_string.split(",")]
     output_components = []
+
     for c in components:
+        if len(c) == 0:
```

Comment:
Not that it matters much here, but I think `not c` is faster than `len(c) == 0`

Reply:
but less explicit

Reply:
fair :wink: